man/systemd.netdev.xml

```diff
@@ -2467,7 +2467,7 @@
           <para>Takes a boolean. Specifies that duplicate frames (received on inactive ports)
           should be dropped when false, or delivered when true. Normally, bonding will drop
           duplicate frames (received on inactive ports), which is desirable for
-          most users. But there are some times it is nice to allow duplicate
+          most users. But there are sometimes it is nice to allow duplicate
```

Comment:
"Sometimes" is wrong, but "some times" also sounds weird. Better to rephrase it to either just "times" or "some cases/scenarios"